src/main/java/org/gridsuite/study/server/StudyService.java

```diff
@@ -430,11 +430,11 @@ private String buildTombstonedEquipmentSearchQuery(UUID networkUuid, String vari
                 networkModificationTreeService.doDeleteTree(studyUuid);
                 studyRepository.deleteById(studyUuid);
                 studyInfosService.deleteByUuid(studyUuid);
-                emitStudiesChanged(studyUuid, userId);
+                emitStudyDelete(studyUuid, userId);
             });
         } else {
             studyCreationRequestRepository.deleteById(studyCreationRequestEntity.get().getId());
-            emitStudiesChanged(studyUuid, userId);
+            emitStudyDelete(studyUuid, userId);
         }
```

Comment:
move outside the if else block to factorize ?

Reply:
Done